forge-ai/src/main/java/forge/ai/ComputerUtilCard.java

```diff
@@ -1406,6 +1406,39 @@ && doesCreatureAttackAI(ai, c)) {
             }
         }
 
+        if (keywords.contains("Banding") && !c.hasKeyword(Keyword.BANDING)) {
+            if (phase.is(PhaseType.COMBAT_BEGIN) && phase.isPlayerTurn(ai)) {
+                // will this card participate in an attacking band?
+                Card bandingCard = getPumpedCreature(ai, sa, c, toughness, power, keywords);
+                // TODO: It may be possible to use AiController.getPredictedCombat here, but that makes it difficult to
+                // use reinforceWithBanding through the attack controller, especially with the extra card parameter in mind
+                AiAttackController aiAtk = new AiAttackController(ai);
+                Combat predicted = new Combat(ai);
+                aiAtk.declareAttackers(predicted);
+                aiAtk.reinforceWithBanding(ai, predicted, bandingCard);
+                if (predicted.isAttacking(bandingCard) && predicted.getBandOfAttacker(bandingCard).getAttackers().size() > 1) {
```

Comment:
Hmn, what if there is only attacker without banding -
Would the AI think it could then band with the virtual copy of itself?